pages/guide.py

```diff
@@ -283,6 +283,8 @@ def process_post_data(self, feature_id, stage_id=0):
       feature = models.Feature.get_by_id(feature_id)
       if feature is None:
         self.abort(404, msg='Feature not found')
+      else:
+        feature.stash_values()
 
```

Comment:
Calling the stash_values() here as mentioned [here](https://github.com/GoogleChrome/chromium-dashboard/pull/1365#issuecomment-864134466). Thank you for letting me know there was only one place where this method call was required.